### PR TITLE
fixed missing dash in xiana.swagger

### PR DESCRIPTION
**Pull request by patbrown**
- get-apidoc is -get-apidoc

### Description
Says it all. There was a dash missing from -get-apidocs. If you want me to put something on the changelog I'll resubmit. I'll leave it up to you if you want to track such minor things. 

### Xiana
I'm enjoying how minimalistic what you are doing is. Thanks for the open source.

### CODE REVIEW

**Comment by soulflyer** [comment]
-get-apidocs refers to the Protocol but we are calling the function which is `get-apidocs`. Note that we are using version 0.5.18 not the most recent version which doesn't appear to have the fn, only the protocol. That is something we will need to sort out if and when we update the reitit version.

File: src/xiana/webserver.clj [hunk 1]
```diff
@@ -1,7 +1,7 @@
 (ns xiana.webserver
   "Lifecycle management of the webserver"
   (:require
-    [ring.adapter.jetty9 :as jetty]
+    [org.httpkit.server :as server]
```
Comment (soulflyer) [inline src/xiana/webserver.clj:4]: Why do we want to change to httpkit?

**Comment by soulflyer** [comment]
There appear to be a number of things happening in this PR. We really need to stick to one issue per PR.

**Comment by patbrown** [comment]
Oh geez, please forgive me. I was late to see this. I really messed up this pull request. I only wanted to PR fixing the dash. SORRY!

**Pull request closed without merging**